### ISSUE TITLE
Error thrown with hashbang url's (0.0.10)

### ISSUE THREAD

**Issue by serkandurusoy** [okgrow/router-autoscroll#1]
The _first time_ I visit hashbang url's, I get the following error in the console. When I hit reload with that url, the error disappears.

```
Exception from Tracker afterFlush function: debug.js:41
Error: Syntax error, unrecognized expression: #/reset-password/jg_zGyhYUqdKijbyKgSKA1-bqWa7MBate4fjJ_F2Dgm
    at Function.Sizzle.error (jquery.js:1491)
    at Sizzle.tokenize (jquery.js:2108)
    at Sizzle.select (jquery.js:2512)
    at Function.Sizzle [as find] (jquery.js:888)
    at jQuery.fn.extend.find (jquery.js:2728)
    at jQuery.fn.init (jquery.js:2845)
    at jQuery (jquery.js:73)
    at router-autoscroll.js:46
    at Object.Tracker._runFlush (tracker.js:468)
    at Object.Tracker.flush (tracker.js:412)
```

It seems to be pointing at line 46 of router-autoscroll.js

**Comment by deanrad** [comment]
Yes, I see what that's about and I can push something for that today.

**Comment by serkandurusoy** [comment]
:+1:

**Comment by deanrad** [comment]
For the record, are you using iron:router or flow-router ?

**Comment by serkandurusoy** [comment]
Oh, yes, sorry about that. I'm using flow-router!

**Comment by deanrad** [comment]
I published a new version, 0.0.11 - it will not console error any more. Please confirm.

**Comment by serkandurusoy** [comment]
Yes, I confirm this. Thank you!
